avocado/utils/asset.py

```diff
@@ -40,6 +40,10 @@
 log = logging.getLogger('avocado.test')
 
 
+#: The default hash algorithm to use on asset cache operations
+DEFAULT_HASH_ALGORITHM = 'sha256'
```

Comment:
As I mentioned on meeting, improving the security hash algorithm won't make us secure and using (nowadays) secure algorithm by default might give the impression this is meant for security. Anyway our code does not really care about security, it's for caching purposes only and very simple, therefor I'd rather see the fast (usually in HW sha1) here and users that want security should use other means to double-check what they downloaded (like using two hash function one by us and other by their code).

Anyway this is not a nack, just my concern. Speed difference on my machine is 1:2.2, which is quite significant given the benefits ...

Reply:
I think this can be left for a later moment.  Specially because with this change, not only cache validation will fail, but so will failure to fetch assets on tests that don't explicitly define the algorithm.  So, to avoid users having tests suddenly FAILing, let's skip this for now.